crates/pathfinder/src/core.rs

```diff
@@ -269,17 +357,10 @@ pub enum BlockId {
 }
 
 impl StarknetBlockNumber {
-    pub const GENESIS: StarknetBlockNumber = StarknetBlockNumber(0);
+    pub const GENESIS: StarknetBlockNumber = StarknetBlockNumber::new_or_panic(0);
     /// The maximum [StarknetBlockNumber] we can support. Restricted to `u64::MAX/2` to
     /// match Sqlite's maximum integer value.
-    #[cfg(test)]
-    pub const MAX: StarknetBlockNumber = StarknetBlockNumber(i64::MAX as u64);
-}
-
-impl std::cmp::PartialOrd for StarknetBlockNumber {
-    fn partial_cmp(&self, other: &Self) -> Option<std::cmp::Ordering> {
-        self.0.partial_cmp(&other.0)
-    }
```

Comment:
I couldn't figure out why would this need to be manual so I got rid of it.

Reply:
Fairly sure it was just me being dumb :)

Reply:
Oki good